Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -495,21 +495,71 @@ void SpreadsheetWidget::change_cell_static_color_format(Spreadsheet::FormatType
 {
     VERIFY(current_worksheet_if_available());
 
-    auto dialog = GUI::ColorPicker::construct(Color::White, window(), "Select Color");
-    if (dialog->exec() == GUI::Dialog::ExecResult::OK) {
+    auto preview_color_in_selected_cells = [this, format_type](Gfx::Color color) {
+        for (auto& position : current_worksheet_if_available()->selected_cells()) {
+            auto* cell = current_worksheet_if_available()->at(position);
+            auto previous_type_metadata = cell->type_metadata();
+            if (format_type == Spreadsheet::FormatType::Background)
+                cell->type_metadata().static_format.background_color = color;
+            else
+                cell->type_metadata().static_format.foreground_color = color;
+            update();
+        }
+        window()->set_modified(true);
```

Comment:
You probably don't want to set the window as in the preview:
```suggestion
```

Reply:
Fixed!